tests/components/samsungtv/test_config_flow.py

```diff
@@ -581,12 +589,13 @@ async def test_ssdp_already_configured(hass: HomeAssistant, remote: Mock):
         assert entry.unique_id == "0d1cef00-00dc-1000-9c80-4844f7b172de"
 
 
-async def test_import_legacy(hass: HomeAssistant, remote: Mock):
+async def test_import_legacy(hass: HomeAssistant, remote: Mock, no_mac_address: Mock):
     """Test importing from yaml with hostname."""
     with patch(
         "homeassistant.components.samsungtv.config_flow.socket.gethostbyname",
         return_value="fake_host",
-    ), patch("getmac.get_mac_address", return_value="aa:bb:cc:dd:ee:ff"):
+    ):
+        no_mac_address.return_value = "aa:bb:cc:dd:ee:ff"
```

Comment:
I'd move this up out of the patch context manager.

Reply:
Done